src/hal/src/queue/mod.rs

```diff
@@ -1,12 +1,11 @@
-/*! Command queues.
-
-    Queues are the execution paths of the graphical processing units. These process
-    submitted commands buffers.
-
-    There are different types of queues, which can only handle associated command buffers.
-    `CommandQueue<B, C>` has the capability defined by `C` for backend `B`: graphics, 
-    compute and transfer.
-!*/
+//! Command queues.
```

Comment:
is that a new standard - to use single-line comments for the module docs?

Reply:
It's just what I've seen everywhere else...